commands/help.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 bot_name = os.getenv("bot_name")
+bot_name.replace("_","\\_")
```

Comment:
Now that the parse mode is in HTML, do we need to escape this `_`?

Reply:
yes it was not needed